lib/web_ui/lib/src/engine/window.dart

```diff
@@ -45,6 +46,17 @@ base class EngineFlutterView implements ui.FlutterView {
     EnginePlatformDispatcher platformDispatcher,
   ) = _EngineFlutterViewImpl;
 
+  factory EngineFlutterView.implicit(
```

Comment:
Should `EngineFlutterView.implicit` not take a `viewId`? It seems we assert in some places that the `viewId` for the implicit view should be `kImplicitViewId`

Reply:
Definitely